### PR TITLE
Docs specviz

### PR DESCRIPTION
**Pull request by camipacifici**
### Description

This pull request is to update the docs for specviz display and plugins. I had to sneak in a couple of updates also in the imviz docs.

[Rendered docs](https://jdaviz--2727.org.readthedocs.build/en/2727/)

### Change log entry

- [ ] Is a change log needed? If yes, is it added to `CHANGES.rst`? If you want to avoid merge conflicts,
  list the proposed change log here for review and add to `CHANGES.rst` before merge. If no, maintainer
  should add a `no-changelog-entry-needed` label.

### Checklist for package maintainer(s)

This checklist is meant to remind the package maintainer(s) who will review this pull request of some common things to look for. This list is not exhaustive.

- [x] Are two approvals required? Branch protection rule does not check for the second approval. If a second approval is not necessary, please apply the `trivial` label.
- [ ] Do the proposed changes actually accomplish desired goals? Also manually run the affected example notebooks, if necessary.
- [ ] Do the proposed changes follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [ ] Are tests added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Are docs added/updated as required? If so, do they follow the [STScI Style Guides](https://github.com/spacetelescope/style-guides)?
- [x] Did the CI pass? If not, are the failures related?
- [x] Is a milestone set? Set this to bugfix milestone if this is a bug fix and needs to be released ASAP; otherwise, set this to the next major release milestone. Bugfix milestone also needs an accompanying backport label.
- [ ] After merge, any internal documentations need updating (e.g., JIRA, Innerspace)?

### CODE REVIEW

File: docs/imviz/img/plugins.jpg
Comment (gibsongreen) [inline docs/imviz/img/plugins.jpg:1]: I see that this file was added to Imviz images directory. In my local build for this branch, Imviz's Data Analysis Plugins tab is displaying the doc/img/plugins.jpg file here. Is this correct, or should these files be switched out?

Comment (camipacifici) [reply docs/imviz/img/plugins.jpg:1]: I might have messed up the place for this image. When updating the equivalent for specviz, I realized that the one for imviz was also old (did not have "Footprints"). I will clean this up. Thanks

Comment (camipacifici) [reply docs/imviz/img/plugins.jpg:1]: Should be fixed now. I checked that the old plugin figure was not used and deleted it.

File: docs/specviz/plugins.rst [hunk 1]
```diff
@@ -101,10 +104,37 @@ or to change to subtraction, for example.
 After fitting, the expandable menu for each component model will update to
 show the fitted value of each parameter rather than the initial value, and
 will additionally show the standard deviation uncertainty of the fitted
-parameter value if the parameter was not set to be fixed to the initial value.
+parameter value if the parameter was not set to be fixed to the initial value
+and if the spectrum uncertainty was loaded.
 
-Parameter values for each fitting run are stored in the plugin table.  
-To export the table into the notebook via the API, call
+Form the API
```
Comment (rosteen) [inline docs/specviz/plugins.rst:110]:
```suggestion
From the API
```

File: docs/specviz/plugins.rst [hunk 2]
```diff
@@ -101,10 +104,37 @@ or to change to subtraction, for example.
...
+------------
+
+The model fitting plugin can be run from the API. Here is an example.
```
Comment (rosteen) [inline docs/specviz/plugins.rst:113]:
```suggestion
The model fitting plugin can be run from the API:
```

**Review by rosteen — COMMENTED** [review]
Two minor suggestions but otherwise looks good to me.

**Comment by camipacifici** [comment]
> Two minor suggestions but otherwise looks good to me.

Suggestions taken, thank you!

**Review by rosteen — APPROVED** [review]

**Review by gibsongreen — APPROVED** [review]
All looks good to me too! 💯

File: docs/specviz/displaying.rst [hunk 3]
```diff
@@ -90,16 +94,6 @@ Interactive Pan/Zoom (Desktop or Notebook Interface)
 You can find the following Pan/Zoom tools available in the viewer toolbar
 on the top right of the viewer.
 
-
-Home
```
Comment (pllim) [inline docs/specviz/displaying.rst:94]: This was completely removed? Was it on purpose?

Comment (rosteen) [reply docs/specviz/displaying.rst:94]: It looks like it was a duplicate section, you can see above there's a home heading with the same text.

**Pull request merged**